incubator/mysqlha/templates/statefulset.yaml

```diff
@@ -223,21 +223,30 @@ spec:
           name: {{ template "fullname" . }}
       - name: scripts
         emptyDir: {}
-      {{- if eq .Values.persistence.enabled false }}
+{{- if not .Values.persistence.enabled }}
```

Comment:
Don't use `not` and swap the contents of the if/else blocks.

Reply:
Why should not `not` be used?

Reply:
Now swapped